sml_small/editing/thousand_pounds/thousand_pounds.py

```diff
@@ -267,17 +299,47 @@ def validate_input(
     )
 
 
-def determine_tpc_marker(do_adjustment: bool) -> str:
+def check_zero_errors(
+    predictive: Optional[float], auxiliary: Optional[float]
+) -> TpcMarker:
+    """
+    Checks predictive and auxiliary to ensure that there is not only a 0 value available,
+    as this will cause a divide by 0 error
+
+    :param predictive: Value used for 'previous' response (Returned/Imputed/Constructed)
+    :type predictive: Optional[float]
+    :param auxiliary: Calculated response for the 'previous' period
+    :type auxiliary: Optional[float]
+
+    :return: tpc_marker, either method_proceed or stop
+    :rtype: TpcMarker
+    """
+    tpc_marker = TpcMarker.METHOD_PROCEED
+    if predictive == 0 and (auxiliary is None or auxiliary == 0):
```

Comment:
What happens if predictive is None and auxiliary is 0?

Reply:
amended to also set stop if predictive is None, spoke to Ellie and she agrees that it should be a scenario that outputs a stop value and will update the test data next week